src/constraints/constraint_storage_line_pack.jl

```diff
@@ -0,0 +1,78 @@
+#############################################################################
+# Copyright (C) 2017 - 2018  Spine Project
+#
+# This file is part of SpineOpt.
+#
+# SpineOpt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SpineOpt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+"""
+    add_constraint_storage_line_pack!(m::Model)
+
+Constraint for line storage dependent on line pack.
+"""
+function add_constraint_storage_line_pack!(m::Model)
+    @fetch node_state, node_pressure = m.ext[:variables]
+    m.ext[:constraints][:storage_line_pack] = Dict(
+    (connection=conn,node1=stor,node2=ng, stochastic_path=s, t=t) => @constraint(
+        m,
+        sum(
+        node_state[stor,s,t]*duration(t)
+        for (stor,s,t) in node_state_indices(m;node=stor,stochastic_scenario=s,t=t_in_t(m; t_long=t))
+        )
+        ==
+        connection_linepack_constant(connection=conn,node1=stor,node2=ng)
+        #connection_linepack_constant[(connection=conn,node1=stor,node2=ng,stochastic_scenario=s,analysis_time=t0, t=t)] #TODO: fails for some reason

```

Comment:
Do we need to fix this before merging?

Reply:
I feel like this might be an issue in general - I haven't quite figured it out. My suspicion is, that it has something to do with groups + stochastic parameter call. I'll open an issue on it

Reply:
Sounds good, I can investigate (even better if you can provide a quick example).